public/app/features/dashboard/containers/DashboardPage.tsx

```diff
@@ -392,7 +392,12 @@ export class UnthemedDashboardPage extends PureComponent<Props, State> {
             </section>
           )}
 
-          <DashboardGrid dashboard={dashboard} viewPanel={viewPanel} editPanel={editPanel} />
+          <DashboardGrid
+            dashboard={dashboard}
+            isEditable={!!dashboard.meta.canEdit}
```

Comment:
Probably you can use the model-getter to make sure we can abstract the logic behind editing a dashboard.
```suggestion
            isEditable={dashboard.canEditDashboard()}
```